lib/LightPanel/LightPanel.js

```diff
@@ -149,12 +149,23 @@ module.exports = kind(
 		if (!this.$.client.children.length) {
 			this.createComponents(this.clientComponents);
 			this.$.client.render();
-			this.$.client.addClass('populated');
+			this.didClientRender();
 		}
 
-		if (!Spotlight.getCurrent()) {
+		if (!Spotlight.getPointerMode() && !Spotlight.isSpottable(Spotlight.getCurrent(), true)) {
```

Comment:
Hi @KunmyonChoi I've gone a different direction and refactored via this PR (https://github.com/enyojs/moonstone/pull/2277) to address Spotlight issues in a more consistent way, so I don't think we will end up needing this specific change.